flexget/plugins/list/trakt_list.py

```diff
@@ -245,9 +245,12 @@ def invalidate_cache(self):
         self._items = None
 
     def get_list_endpoint(self, remove=False, submit=False):
-        # Api restriction, but we could easily extract season and episode info from the 'shows' type
         if not submit and self.config['list'] in ['collection', 'watched'] and self.config['type'] == 'episodes':
-            raise plugin.PluginError('`type` cannot be `%s` for %s list.' % (self.config['type'], self.config['list']))
+            # API restriction as they don't have an endpoint for collected episodes yet
+            if self.config['list'] == 'collection':
+                raise plugin.PluginError('`type` cannot be `%s` for %s list.' % (self.config['type'], self.config['list']))
```

Comment:
There's no longer any reason for this string interpolation business. It could just be changed to 
```
'`type` cannot be `episodes` for collection list.'`
```

You also don't really need the `else` as execution will only reach either the `raise` or the `return`.

Reply:
Cleaned it up now, hope it's fine now.